### ISSUE TITLE
Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease  Unable to find expected entry 'main/binary-amd64/Packages'

### ISSUE THREAD

**Issue by hoanphi2201** [judge0/api#84]
![image](https://user-images.githubusercontent.com/36753744/58304826-1ea9ab80-7e20-11e9-8096-43b53cec7476.png)

Hello. What do I have to do to fix it?

**Comment by gargpk** [comment]
hey @hoanphi2201 

I'm too facing this problem.

found this solution:
https://unix.stackexchange.com/a/508728
can you take a look at this and revert if this worked for you.

**Comment by hoanphi2201** [comment]
@gargpk 
Can you show me how to do it?
![image](https://user-images.githubusercontent.com/36753744/58311862-30954980-7e34-11e9-99d4-59e9c35604be.png)

**Comment by hoanphi2201** [comment]
Did you fix it successfully?

**Comment by gargpk** [comment]
@hoanphi2201  
i tried but was not successful

@hoanphi2201 
i did this as follows:

adding these lines of code in the Dockerfile.dev just after LABEL  

```
RUN echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
#RUN echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie main" > /etc/apt/sources.list.d/jessie.list

RUN sed -i '/deb http:\/\/deb.debian.org\/debian jessie-updates main/d' /etc/apt/sources.list

RUN apt-get -o Acquire::Check-Valid-Until=false update
```

you can check this thread for more details regarding this issue:
[stackexchange](https://unix.stackexchange.com/a/508728)
and this one 
[stackoverflow](https://stackoverflow.com/a/55411551[)

**Comment by hoanphi2201** [comment]
@gargpk 
 I also tried it but failed. It took me a long time to find a solution. 
I don't know what I should do right now

**Comment by gargpk** [comment]
@hoanphi2201 

i read alot of threads they are suggesting to migrate to debian-stretch  from debian-jessie as this is officially deprecated 

they are many workaround tried by people, many of them i also tried, none is working.
i think we have to create a docker image for debian-stretch compatible with judge0/api

**Comment by hoanphi2201** [comment]
It will be great if you solve it. I wait for a solution from you.

**Comment by hoanphi2201** [comment]
Do you encounter this problem? When I use https://api.judge0.com/ on the postman it returns stdout, when it returns null in the application

**Comment by gargpk** [comment]
have you checked this:
https://api.judge0.com/dummy-client.html
this is working completely fine and so is the:
https://github.com/judge0/api/releases/tag/v1.1.5
in release mode maybe containing all the details and all try this one may be a relief.

**Comment by hoanphi2201** [comment]
This version can build on my device, right?

**Comment by hoanphi2201** [comment]
I have seen it work

**Comment by hoanphi2201** [comment]
`````
#include <iostream>

long unsigned int fib(unsigned int n);

long unsigned int fib(unsigned int n) {
	if (n == 0) return 0;
	if (n == 1) return 1;
	return fib(n-1) + fib(n-2);
}

int main() {
	unsigned int number;

	while (std::cin >> number) {
		std::cout << fib(number) << std::endl;
	}
	return 0;
}

`````
You can test it: language_id: 10

**Comment by gargpk** [comment]
@hoanphi2201  
its working completely fine here is the output 
```
[Request 5/24/2019, 3:47:59 PM] POST http://192.168.1.174:3000/submissions?wait=true
{
    "source_code": "#include <iostream>\n\nlong unsigned int fib(unsigned int n);\n\nlong unsigned int fib(unsigned int n) {\n\tif (n == 0) return 0;\n\tif (n == 1) return 1;\n\treturn fib(n-1) + fib(n-2);\n}\n\nint main() {\n\tunsigned int number;\n\n\twhile (std::cin >> number) {\n\t\tstd::cout << fib(number) << std::endl;\n\t}\n\treturn 0;\n}\n",
    "language_id": "10",
    "number_of_runs": "1",
    "stdin": "5",
    "expected_output": "5",
    "cpu_time_limit": "2",
    "cpu_extra_time": "0.5",
    "wall_time_limit": "5",
    "memory_limit": "128000",
    "stack_limit": "64000",
    "max_processes_and_or_threads": "30",
    "enable_per_process_and_thread_time_limit": false,
    "enable_per_process_and_thread_memory_limit": true,
    "max_file_size": "1024"
}
[Response 5/24/2019, 3:47:59 PM] 201 Created
{
    "stdout": "5\n",
    "time": "0.004",
    "memory": 496,
    "stderr": null,
    "token": "51c108af-d33d-4c3b-80a9-a5cabef00e9a",
    "compile_output": null,
    "message": null,
    "status": {
        "id": 3,
        "description": "Accepted"
    }
}

[DONE 5/24/2019, 3:47:59 PM]
```

**Comment by hoanphi2201** [comment]
Oh, tôi đã phát hiện ra vấn đề ở đây. Tôi cần setTimeOut cho qúa trình lấy dữ liệu trả về. Có phải tôi đã hiểu đúng?

**Comment by hoanphi2201** [comment]
Oh, I discovered the problem here. I need setTimeOut for the process of retrieving the returned data. Did I understand correctly?

**Comment by hoanphi2201** [comment]
Thank you for your concern. I have done it. we should have a callback when status.description == processing || in queue.

**Comment by hoanphi2201** [comment]
@gargpk 
Can I send multiple test sets in one submit?

**Comment by gargpk** [comment]
I don't think so.
we can't!

**Comment by gargpk** [comment]
this apis production environment is working fine but its dev environment is broken need to fix it soon.

**Comment by hoanphi2201** [comment]
@gargpk  413 Request Entity Too Large
Have you met this situation? I need a solution to do it with big test kits

**Comment by hermanzdosilovic** [comment]
Hi everybody. I am aware of this problem with buildpack-deps:jessie image and I am working on it.

**Comment by pethanin** [comment]
Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease Unable to find expected entry 'main/binary-amd64/Packages' - facing same issue.

**Comment by hermanzdosilovic** [comment]
This is fixed with [Judge0 API v1.2.0](https://github.com/judge0/api/releases/tag/v1.2.0).
